vulture/core.py

```diff
@@ -255,6 +255,29 @@ def by_size(item):
         return sorted(confidently_unused,
                       key=by_size if sort_by_size else by_name)
 
+    def make_whitelist(self, min_confidence=0, sort_by_size=False):
+        unused_code = self.get_unused_code(
+                min_confidence=min_confidence, sort_by_size=sort_by_size)
+        if any(item.typ in ('attribute', 'property') for item in unused_code):
```

Comment:
Could you please use lists instead of tuples for homogeneous collections (here and below)? I.e., square brackets instead of normal brackets.

Reply:
I am sorry, I'm a little confused 😅. I thought that we were gonna [remove this part](https://github.com/jendrikseipp/vulture/pull/135#discussion_r199561968) because we didn't need `VultureMock` class?

Reply:
Right, I should have placed the comment somewhere else, e.g., I prefer `['function', 'class', 'variable', 'import']` instead of `('function', 'class', 'variable', 'import')`, etc.

Reply:
Changes done, Thanks for the clarification. :-)